Library/Homebrew/cmd/tap.rb

```diff
@@ -27,11 +27,6 @@ def tap_args
         assumptions, so taps can be cloned from places other than GitHub and
         using protocols other than HTTPS, e.g. SSH, git, HTTP, FTP(S), rsync.
       EOS
-      switch "--full",
-             description: "Convert a shallow clone to a full clone without untapping. Taps are only cloned as "\
-                          "shallow clones if `--shallow` was originally passed."
-      switch "--shallow",
-             description: "Fetch tap as a shallow clone rather than a full clone. Useful for continuous integration."
```

Comment:
Perhaps we should do this automatically if `ENV["CI"]` is set?

Reply:
If you mean fetch the tap as a shallow clone, I agree. I can make this change.

Reply:
The full `git` history is needed for commands like `brew audit`, right? Assuming yes, I don't think we should do this.

If a user _must_ have a shallow clone, they can always manually clone into the correct directory, so I don't think we need to provide the option.

Reply:
Fine with me.